pkg/build/api/types.go

```diff
@@ -512,7 +512,7 @@ type SourceBuildStrategy struct {
 	Scripts string
 
 	// Incremental flag forces the Source build to do incremental builds if true.
-	Incremental bool
+	Incremental *bool
```

Comment:
This definitely changes behavior.  Before, if I didn't specify this field in my BuildConfig all the Builds it created would be `false`.  Now, if I don't specify the value it can sometimes be `true`.  I don't necessarily object to the change, but if `incremental` doesn't work all the time, I'd end up with buildconfigs that used to work doing new and different things.